product_docs/docs/pgd/5/overview/index.mdx

```diff
@@ -23,7 +23,7 @@ Each node (database) participating in a PGD group both receives changes from oth
 
 This is distinct from hot or warm standby, where only one master server accepts writes and all the other nodes are standbys that replicate either from the master or from another standby.
 
-You don't have to write to all the masters all of the time. A frequent configuration directs writes mostly to just one master called the [write leader](../terminology/#write-leader).
+You don't have to write to all the masters all of the time. A frequent configuration directly writes mostly to just one master called the [write leader](../terminology/#write-leader).
```

Comment:
No, that should be directs as in directs traffic.

Reply:
oh jeez, I just got that read. Apologies!

Reply:
I changed it back.